### DISCUSSION TITLE
I am new user of trame

### DISCUSSION THREAD

**Discussion by yjtwinpark** [Kitware/trame#206] [Q&A] [answered]
I'm willing to use trame for my research web, so now I follow step tutorial.

From overview to application, there was no problem (kitware.github.io/trame/docs/tutorial-application.html)
but I experience some problem in paraview tutorial.

Even I installed trame and paraview5.11 version, 
there was error message "moduleNotFoundError No module name paraview"
Maybe do I have to do setting something, but I dont know explanation in paraview tutorial script.
(in fact, im newbie for not just trame but also python and English as well)

Can you help me?

**Comment by jourdain** [comment, marked as answer]
So for ParaView, you need to run your script from `paraview/bin/pvpython` and provide information on a compatible virtual-environment (Python 3.9) so ParaView can have access to trame. 

This is capture in that [README](https://github.com/Kitware/trame/tree/master/examples/07_paraview)

**Reply by yjtwinpark** [reply]
now I understood! I did't know I have to run pvpython file with arg. And I was confused with "Running an example" in tutorial paraview, explanation of "\" means just line break.
